src/components/ec-text-field/ec-text-field.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="ec-text-field">
+    <div
+      v-if="label"
+      class="ec-text-field__label"
+    >{{ label }}</div>
+    <input
```

Comment:
We need to add `v-bind="$attrs" v-on="$listeners"` here because currently we cannot do this:
`<ec-text-field type="number" min="5" max="10" @focus="myHandler()" />`. We need to write a test for it too.